Xamarin.Forms.Platform.iOS/PlatformRenderer.cs

```diff
@@ -91,7 +91,7 @@ public override void ViewDidLoad()
 		{
 			base.ViewDidLoad();
 			SetNeedsStatusBarAppearanceUpdate();
-			if (Forms.IsiOS11OrNewer)
+			if (RespondsToSelector(new ObjCRuntime.Selector("setNeedsUpdateOfHomeIndicatorAutoHidden")))
```

Comment:
The casing for the selector is different in this instance than the others.

I'm wondering if it would make sense to move this into an extension method for UIViewController which does the check and calls `SetNeedsUpdateOfHomeIndicatorAutoHidden()` as appropriate. Rather than duplicating this code in several places.